config/application.rb

```diff
@@ -28,7 +28,13 @@ class Application < Rails::Application
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
 
-    config.autoload_paths << Rails.root.join('lib')
+    # config.autoload_paths << Rails.root.join('lib')
```

Comment:
With the bulk of the work in `app/domains/...`, do we still need these changes?

Reply:
Don't think it's needed for my code anymore.  Tomorrow I'll try pulling it and rerun the cukes.